docs/forecasting.md

```diff
@@ -0,0 +1,26 @@
+# Forcasting
+
+Numalogic supports the following variants of forecasting.
+
+### Baseline Forecaster
+
+```python
+from numalogic.models.forecast.variants import BaselineForecaster
+
+model = BaselineForecaster()
+model.fit(train_df)
+pred_df = model.predict(test_df)
+r2_score = model.r2_score(test_df)
+anomaly_score = model.score(test_df)
+```
+### Seasonal Naive Forecaster
+
```

Comment:
A naive forecaster that takes seasonality into consideration and predicts the previous day/week values.